src/components/MultiHopTrade/components/TradeInput/components/QuoteTimer.tsx

```diff
@@ -0,0 +1,50 @@
+import { CircularProgress } from '@chakra-ui/react'
```

Comment:
This is the wrong import as compared to the previous version, which is why styles are borked. Should be 
```suggestion
import { CircularProgress } from '@/components/CircularProgress/CircularProgress'
````

Reply:
Great spotting thanks for that. Fixed here https://github.com/shapeshift/web/pull/9806/commits/513560ca4dd61dc7b0f77da95ecbaea60cddd563